fence/blueprints/login/ras.py

```diff
@@ -133,29 +133,40 @@ def post_login(self, user=None, token_result=None):
             user=user, refresh_token=refresh_token, expires=expires + issued_time
         )
 
-        usersync = config.get("USERSYNC", {})
-        sync_from_visas = usersync.get("sync_from_visas", False)
-        # Check if user has any project_access from a previous session or from usersync AND if fence is configured to use visas as authZ source
-        # if not do an on-the-fly usersync for this user to give them instant access after logging in through RAS
-        if not user.project_access and sync_from_visas:
-            # Close previous db sessions. Leaving it open causes a race condition where we're viewing user.project_access while trying to update it in usersync
-            # not closing leads to partially updated records
-            current_session.close()
-            DB = os.environ.get("FENCE_DB") or config.get("DB")
-            if DB is None:
-                try:
-                    from fence.settings import DB
-                except ImportError:
-                    pass
-            dbGaP = os.environ.get("dbGaP") or config.get("dbGaP")
-            if not isinstance(dbGaP, list):
-                dbGaP = [dbGaP]
-
-            sync = init_syncer(
-                dbGaP,
-                None,
-                DB,
-            )
-            sync.sync_single_user_visas(user, current_session)
+        global_parse_visas_on_login = config["GLOBAL_PARSE_VISAS_ON_LOGIN"]
+
+        if global_parse_visas_on_login or global_parse_visas_on_login == None:
+            parse_visas = query_params.get("parse_visas")
```

Comment:
```suggestion
            parse_visas = "parse_visas" in query_params
```
i think right now just `?parse_visas` (no value) would be interpreted as False by `if parse_visas`. Since there is no documentation for this param yet, i'm not sure that's on purpose.

Also I kinda disagree with your and @Avantol13's discussion above - i think if we expect a value, we should handle `?parse_visas=false` properly.

tldr; choose whether we accept `?parse_visas`, or `?parse_visas=true/false`, or both, and update the doc and the code accordingly